cmd/kubeadm/app/preflight/checks.go

```diff
@@ -182,6 +184,38 @@ func (fac FileAvailableCheck) Check() (warnings, errors []error) {
 	return nil, errors
 }
 
+// FileContentCheck checks that the given file contains the string Content.
+type FileContentCheck struct {
+	Path    string
+	Content []byte
+}
+
+func (fcc FileContentCheck) Check() (warnings, errors []error) {
```

Comment:
@luxas I had misread your comment. What @dmmcquay states is correct as, in principle, you should use pointer receiver only when you want to modify the receiver or the struct is large and deep copy is expensive. Also, if we're looking for consistency, other checks don't use pointers.